react/features/base/conference/functions.js

```diff
@@ -298,6 +300,42 @@ export function getRoomName(state: Object): string {
     return getConferenceState(state).room;
 }
 
+/**
+ * Get an obfuscated room name or create and persist it if it doesn't exists.
+ *
+ * @param {Object} state - The current state of the app.
+ * @param {Function} dispatch - The Redux dispatch function.
+ * @returns {string} - Obfuscated room name.
+ */
+export function getOrCreateObfuscatedRoomName(state: Object, dispatch: Function) {
```

Comment:
Sorry I missed this earlier.

This is not going to work on mobile.

On mobile the state is not cleared implicitly between conferences. So the 2nd time you join a meeting you'd be using the obfuscated name of the first one.

Instead of storing this on Redux you could attach it to the JitsiConference with a Symbol defined here so it's only readable for this feature.

Reply:
I think that would be a bit dangerous because if we make the mistake of sending any event in between it would be incorrect.

On mobile it's possible to be disconnecting from one meeting while connecting to another one for example.

Reply:
Understood, will change accordingly.

Reply:
It seems that the JitisiConference Object doesn't get set at this point, I've added a different approach.

Reply:
I like it!